contracts/modules/SuperMinterV1_1.sol

```diff
@@ -393,6 +410,31 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
         l.unitPrice = f.unitPrice;
 
         emit Minted(p.edition, p.tier, p.scheduleNum, p.to, l, p.attributionId);
+
+        return l.fromTokenId;
+    }
+
+    /**
+     * @inheritdoc ISuperMinterV1_1
+     */
+    function platformAirdrop(PlatformAirdrop calldata p) public returns (uint256 fromTokenId) {
+        MintData storage d = _getMintData(LibOps.packId(p.edition, p.tier, p.scheduleNum));
+
+        /* ------------------- CHECKS AND UPDATES ------------------- */
+
+        _requireMintOpen(d);
+
+        if (d.mode != PLATFORM_AIRDROP) revert InvalidMode();
+        _verifyAndClaimPlatfromAidropSignature(d, p);
+
+        _incrementPlatformAirdropMinted(d, p);
+
+        /* ------------------------- MINT --------------------------- */
+
+        ISoundEditionV2_1 edition = ISoundEditionV2_1(p.edition);
+        fromTokenId = edition.airdrop(p.tier, p.to, p.signedQuantity);
```

Comment:
Can only be reached if:
1. the mint is a platform airdrop
2. there is a valid platform signature by the platform specified during mint creation (`createEditionMint` sets the platform that is used to find the signer.
3. the signature hasn't expired
4. the claim ticket number hasn't been used yet
5. the minting doesn't go over the max mintable

As long as this matches the intended restrictions, then this should be good to go.